### PR TITLE
Improve colors in settings 

### PR DESCRIPTION
**Pull request by SimonBrandner**
Type: feature
Improves on https://github.com/vector-im/element-web/issues/19254

<hr>

The changes are a combination of stuff based on designs and of developer design (i.e. me guessing with what compound color could a non-compound color be replaced)

<hr>

List of changes for design to review:

+ [ ] General
	+ [ ] The horizontal separator color
	+ [ ] Subsection text color
	+ [ ] Blue links color
+ [ ] User
	+ [ ] General
		+ [ ] Avatar picker colors
	+ [ ] Appearence
		+ [ ] Theme picker colors
		+ [ ] Layout picker colors
		+ [ ] Preview border color
		+ [ ] Font slider background color
	+ [ ] Security and privacy
		+ [ ] Subheader text color
	+ [ ] Help and about
		+ [ ] Version and access token box border color
		+ [ ] Copy button color
+ [ ] Space
	+ [ ] General 
		+ [ ] Blue link color

---
Here's what your changelog entry will look like:

## ✨ Features
 * Improve colors in settings  ([\#7283](https://github.com/matrix-org/matrix-react-sdk/pull/7283)). Contributed by @SimonBrandner.

Preview: https://pr7283--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/settings/_AvatarSetting.scss [hunk 1]
```diff
@@ -37,7 +37,7 @@ limitations under the License.
         text-align: center;
 
         > span {
-            color: #fff; // hardcoded to contrast with background
+            color: $primary-content; // hardcoded to contrast with background
```
Comment (t3chguy) [inline res/css/views/settings/_AvatarSetting.scss:40]: stale comment

File: res/themes/light/css/_light.scss [hunk 2]
```diff
@@ -113,8 +113,6 @@ $menu-selected-color: #f5f8fa;
 // Settings
 // ********************
 $settings-grey-fg-color: #a2a2a2;
-$settings-profile-overlay-placeholder-fg-color: #2e2f32;
-$settings-profile-button-bg-color: $menu-border-color;
```
Comment (t3chguy) [inline res/themes/light/css/_light.scss:117]: can we not drop these from the other themes too?

File: src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx [hunk 3]
```diff
@@ -357,7 +357,7 @@ export default class SecurityUserSettingsTab extends React.Component<IProps, ISt
                 { warning }
                 <div className="mx_SettingsTab_heading">{ _t("Where you're signed in") }</div>
                 <div className="mx_SettingsTab_section">
-                    <span>
+                    <span className="mx_SettingsTab_subsectionText">
```
Comment (t3chguy) [inline src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx:360]: I think this class is applied to divs elsewhere

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx (hunk 3 above)
Comment (SimonBrandner) [reply src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx:360]: From a quick look, it seems to be applied in most places. The only place where I see it not being applied is `User settings` -> `Preferences` -> `Communities` though I think that was intentional?

**Review by janogarcia — APPROVED** [review]
@SimonBrandner Sorry for the long-overdue review. Thanks for helping out normalizing this. All changes make sense. 👍 The detailed list in the PR description was super helpful!

**Pull request merged**